doc/pic/hld_fpmsyncd.md

```diff
@@ -248,6 +262,32 @@ Therefore, even after this enhancement, table entries will be created for `ROUTE
 ##### Figure: Example of ASIC_DB entry
 ![fig3](images_fpmsyncd/fig3.svg)
 
+#### Orchestration Agent Changes to handle NEXTHOP_GROUP
+
+Orchestration agent `NhgOrch` will be enhanced to handle the new NEXTHOP_GROUP_TABLE in APP_DB. For adding or updating an entry in the NEXTHOP_GROUP_TABLE, programming will depend on whether the group is configured with one or multiple next hops or is a recursive nexthop group i.e. it contains one or more nexthop group.
```

Comment:
Could see code related to handling of Loopback interface and NHID added as CPU port, please capture here if required.

Reply:
This change has been removed from the code PR now